pie.rb

```diff
@@ -0,0 +1,19 @@
+# PIE stands for person in education

```

Comment:
Use the name `PersonInEducation` then.  If I see that you're saying `Student < Pie` I read it as "A `Student` is a `Pie`" which is... not ideal.